src/legacy/core_plugins/kibana/public/visualize_embeddable/visualize_embeddable_factory.tsx

```diff
@@ -23,7 +23,6 @@ import 'uiExports/docViews';
 import 'uiExports/embeddableActions';
 import 'uiExports/fieldFormatEditors';
 import 'uiExports/fieldFormats';
-import 'uiExports/home';
```

Comment:
@maryia-lapata this caused all home ui export entrypoints to be included in all bundles (because the visualize embeddable is loaded as a "hack" entrypoint). Could you confirm that it's not actually needed to render visualizations?

Reply:
I don't think that I'm a right person to confirm that. I just removed these exports during [centralizing dependancies](https://github.com/elastic/kibana/pull/49293/files#diff-53ca92fe2de1ead9193e8c05dc91d415L28) and then returned it back during [Visualize shimming](https://github.com/elastic/kibana/pull/49891/files#diff-53ca92fe2de1ead9193e8c05dc91d415R26). 

I also found that these exports were introduced in https://github.com/elastic/kibana/pull/39126/files#diff-53ca92fe2de1ead9193e8c05dc91d415R29. Maybe it helps.

Reply:
Sorry, just looked one level deep in the git blame :) @stacey-gammon is there a dependency of the visualize embeddable code to the home exports you know of? I'm only seeing feature registrations but maybe I'm missing something.

Reply:
I don't think it's needed, I was probably over-zealous in my copy/pasting that ended that line up here.